Cargo.toml

```diff
@@ -46,6 +46,7 @@ hyper-boring = { version= "2.1.2", features = ['fips'] }
 anyhow = "1.0.65"
 serde_yaml = "0.9.13"
 async-trait = "0.1.58"
+drain = "0.1.1"
```

Comment:
I see this is from linkerd and is probably solid. 

As we add more reviewers (not sure how many will be project-root approvers), we gotta have a well-defined dependency evaluation process.